regression/pages/studio/studio_home.py

```diff
@@ -0,0 +1,23 @@
+from edxapp_pages.studio.index import DashboardPage
+from regression.pages.studio import BASE_URL
+
+
+class DashboardPageExtended(DashboardPage):
+
+    url = BASE_URL + '/home/'
+
+    def is_browser_on_page(self):
+        return self.q(css='.courses-tab.active').present
+
+    def select_course(self, course_title):
+        """
+        Selects the course we want to perform tests on
+        Returns:
+
+        """
+        course_names = self.q(css=course_title)
+
+        for vals in course_names:
+            if 'Manual Smoke Test Course 1 - Auto' in vals.text:
+                vals.click()
+                break
```

Comment:
rather than a `break`, you should have a `wait_for` of some sort to be sure the course has been selected.

Also, I suggest an `else` clause here, in case the course that was requested is not found. Perhaps raise a BrokenPromise error.